### DISCUSSION TITLE
Wii Support?

### DISCUSSION THREAD

**Discussion by Netrider1** [darthcloud/BlueRetro#30] [Q&A] [answered]
I have a WII and would love to use switch pro controllers with it.  Will blueretro work with the wii?

Thanks!

**Comment by darthcloud** [comment, marked as answer]
Well you can use It as a GC controller.

I will add later Wii extension support, but this more targeted at NES/SNES mini classic console.

While nothing could stop you from plugin it in an Wiimote, you would most likely need to power BlueRetro externally. 

But then your inputs would go through two Bluetooth connection which would had an unusual amount of latency.

An interesting idea would be to internaly replace the Wii BT dongle but that dongle interface is USB so not something that will show up in BlueRetro anytime soon unfortunately.

**Comment by Netrider1** [comment]
Thanks for the quick response!
